publish-odr-parameters/README.md

```diff
@@ -37,8 +37,7 @@ Below is an example PR:
 
 ### Step 2a: if you dont want submit workflows
 There may be some cases when you dont want to submit workflows, for example reformatting files.   
-In this case add `[skip ci]` to the pull request title e.g- `fix[skip ci]: Reformatting parameter files`  
-[more information](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/) 
+In this case add `[skip-workflows]` to the pull request title e.g- `fix[skip-workflows]: Reformatting parameter files`  
```

Comment:
`fix[skip-..]:` I think breaks conventional commits format.

How about `fix(something): message [skip-..]`? I wonder if the change log generator is smart enough to remove the `[skip-..]` part